backend/plan/views.py

```diff
@@ -176,6 +183,106 @@ def recommend_courses_view(request):
         status=status.HTTP_200_OK,
     )
 
+class PrimaryScheduleViewSet(viewsets.ViewSet):
```

Comment:
Let's add a docstring to this class with some API documentation for each of the methods (GET, PUT) - same reasoning as for friendships.

"""
list: ...

update: ...
"""